tests/torch/test_export.py

```diff
@@ -0,0 +1,69 @@
+import os
+from pathlib import Path
+
+import pytest
+import torch
+import torchvision.transforms.v2 as T
+import yaml
+from torch.export.pt2_archive._package import load_pt2
+from torchgeo.models import Unet_Weights, unet
+
+from stac_model.torch.export import (
+    export_model_and_transforms,
+    package_model_and_transforms,
+)
+
+
+def export_model(tmpdir: Path, device: str, aoti_compile_and_package: bool) -> None:
+    device = torch.device(device)
+    input_shape = (-1, 8, -1, -1)
```

Comment:
@isaaccorley can we pull this input shape information from the torchgeo datamodule that has logic for how to reshape/crop/resize the model input? so instead of providing an input_shape that is user specified, we provide the datmodule to extract this info.

This test could be updated to not hardcode the shape and instead by default pull this info from a datamodule that works with the TorchGeo Unet model. This could be overridden if the user supplies an input shape.

Reply:
That makes sense to me, how should we allow users to define the shape otherwise though in the case that it's not exported using a torchgeo datamodule?

Reply:
I think we go with an optional `input_shape` arg in that case. if a user tries to export the model with a datamodule and the datamodule doesn't provide shape information, then we throw an error and ask the suer to specify the shape.